rss2email/email.py

```diff
@@ -146,8 +146,11 @@ def get_message(sender, recipient, subject, body, content_type,
         message.set_payload(body, charset=charset)
     if extra_headers:
         for key,value in extra_headers.items():
-            encoding = guess_encoding(value, encodings)
-            message[key] = _Header(value, encoding)
+            if config.getboolean(section, 'encode-date-field') or key != 'Date':
```

Comment:
That’s inverted, the test should be   key == Date  and not <config>

Reply:
The logic is correct. Inverting it would require swapping the true and false branch bodies, is that what you want?

Reply:
Well, the logic is correct but it’s completely unreadable.

I’d rather have one more branch for the Date case and mention `guess_encoding` twice.

```
if == Date:
  if config_set: guess_encoding
  else: dont
else:
  guess_encoding
```